src/components/Radios.test.tsx

```diff
@@ -0,0 +1,280 @@
+        id="error-id"
+      >
+        <span
+          className="govuk-visually-hidden"
+        >
+          My error
+          :
+        </span>
+        There has been an error
+      </span>
+      <div
+        className="govuk-radios lbh-radios test-class govuk-radios--conditional"
+        data-module="govuk-radios"
+      >
+        <div
+          className="govuk-radios__item"
+        >
+          <input
+            aria-describedby="yes-id-item-hint"
+            checked={true}
+            className="govuk-radios__input"
+            data-aria-controls="conditional-yes-id"
+            id="yes-id"
+            name="example"
+            onChange={[Function]}
+            type="radio"
+            value="yes"
+          />
+          <label
+            className="govuk-label lbh-label govuk-radios__label"
+            htmlFor="yes-id"
+          >
+            Yes
+          </label>
+          <span
+            className="govuk-hint lbh-hint govuk-radios__hint yes-hint-class"
+            id="yes-id-item-hint"
+          >
+            Choose this option to select yes
+          </span>
+        </div>
+        <div
+          className="govuk-radios__conditional"
+          id="conditional-yes-id"
+        >
+          <p>
+            Here is some text
+          </p>
+        </div>
+        <div
+          className="govuk-radios__divider"
+        >
+          or
+        </div>
+        <div
+          className="govuk-radios__item"
+        >
+          <input
+            aria-describedby="no-id-item-hint"
+            className="govuk-radios__input"
+            disabled={true}
+            id="no-id"
+            name="example"
+            onChange={[Function]}
+            type="radio"
+            value="no"
+          />
+          <label
+            className="govuk-label lbh-label govuk-radios__label"
+            htmlFor="no-id"
+          >
+            No
+          </label>
+          <span
+            className="govuk-hint lbh-hint govuk-radios__hint no-hint-class"
+            id="no-id-item-hint"
+          >
+            Choose this option to select no
+          </span>
+        </div>
+      </div>
+    </fieldset>
+  </div>
+  `);
+});
+
+it("runs onchange function correctly", () => {
+  const onChange = jest.fn();
+  render(
+    <Radios
+      name="example"
+      className="test-class"
+      items={[
+        {
+          id: "yes-id",
+          label: {
+            children: "Yes"
+          },
+          value: "yes"
+        },
+        {
+          id: "no-id",
+          label: {
+            children: "No"
+          },
+          value: "no"
+        }
+      ]}
+      fieldset={{
+        id: "fieldset-id",
+        className: "fieldset-class",
+        legend: <FieldsetLegend>Have you changed your name?</FieldsetLegend>
+      }}
+      onChange={onChange}
+    />
+  );
+  const label = screen.getByLabelText("Yes");
```

Comment:
`render` returns an object that has these getters. Better to use that than `screen`.